tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -167,7 +183,7 @@ def _test_fsx_lustre_correctly_mounted(remote_command_executor, mount_dir, os, r
     result = remote_command_executor.run_remote_command("df -h -t lustre | tail -n +2 | awk '{print $1, $2, $6}'")
     mount_name = get_mount_name(fsx_fs_id, region)
     assert_that(result.stdout).matches(
```

Comment:
Could you add here an example of the output we're expecting/parsing?

Reply:
Added